cooperts-todo-app/db/migrate/20220512161526_add_not_null_constraint_to_todo_items_done.rb

```diff
@@ -0,0 +1,10 @@
+class AddNotNullConstraintToTodoItemsDone < ActiveRecord::Migration[6.1]
+  def up
+    change_column_null(:todo_items, :done, false)
```

Comment:
I dont know why we need this migration @zheckert

Reply:
@oryolo This was my way to ensure that the boolean checkbox is required and defaulted to false. If we don't need this, we can definitely scrap it!

Reply:
Resolved. Thanks for the help!